app.rb

```diff
@@ -0,0 +1,120 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No books available.'
+    else
+      puts 'All Books:'
+      @books.each do |book|
+        puts "Title: #{book.title} || Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No people available.'
+    else
+      puts 'All People:'
+      @people.each do |person|
+        puts "ID: #{person.id} || Name: #{person.name} || Type: #{person.class}"
+      end
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a Student(1) or a Teacher(2)? [Input the number]: '
+    res = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Age: '
+    age = gets.chomp
+    if res == 1
+      person = Student.new(age, nil, name)
+      puts "Student created successfully. ID: #{person.id}"
+      @people << person
```

Comment:
- [ ] You did a great job setting up the console App. :+1: Let's work on some minor details to make our app similar to the one in the demonstration video. Kindly ask for `parent_permission` when creating a student and use that data when instantiating the Student class. 🤝️